akka-docs/rst/scala/http/routing-dsl/directives/misc-directives/withSizeLimit.rst

```diff
@@ -0,0 +1,40 @@
+.. _-withSizeLimit-:
+
+withSizeLimit
+===============
+
+Signature
+---------
+
+.. includecode2:: /../../akka-http/src/main/scala/akka/http/scaladsl/server/directives/MiscDirectives.scala
+   :snippet: withSizeLimit
+
+Description
+-----------
+Rejects all requests which request entity size exceeds given limit. Limit given
+as parameter overrides limit configured with ``akka.http.parsing.max-content-length``.
```

Comment:
It does not reject, please see https://github.com/akka/akka/pull/20760/commits/9e364c89214d2b6f58c1e72a393836a3e033819d#r68016626